doc/source/whatsnew/v1.2.0.rst

```diff
@@ -464,6 +464,7 @@ I/O
 - Bug in :meth:`read_parquet` with fixed offset timezones. String representation of timezones was not recognized (:issue:`35997`, :issue:`36004`)
 - Bug in output rendering of complex numbers showing too many trailing zeros (:issue:`36799`)
 - Bug in :class:`HDFStore` threw a ``TypeError`` when exporting an empty :class:`DataFrame` with ``datetime64[ns, tz]`` dtypes with a fixed HDF5 store (:issue:`20594`)
+- :class:`TableFormatter` was using the full list of columns when computing formatters for truncated displays (:issue:`35907`)
 
```

Comment:
Currently there is no ``TableFormatter``.
The generic functionality is moved to ``DataFrameFormatter``.